packages/client/test/service/fullethereumservice.spec.ts

```diff
@@ -90,7 +99,7 @@ describe('should open', async () => {
   expect(server.addProtocols).toBeCalled()
   service.config.events.on(Event.SYNC_SYNCHRONIZED, () => {
     it('should synchronize', () => {
-      assert.ok('synchronized')
+      assert.isTrue(true, 'synchronized')
```

Comment:
This whole test structure looks fishy, what if the `it('should syncronize')` never hits - then vitest cant be aware of this "hidden" `it` - right?

Reply:
Also `assert.isTrue(true` is a no-op

Reply:
Agree, never loved these tests but they're hard to rewrite.  Glad there's a potential solution here.